CMakeLists.txt

```diff
@@ -207,6 +207,24 @@ if(${CMAKE_VERSION} VERSION_GREATER "3.6.2")
     endif()
 endif()
 
-# Install MaterialX find_package module
-install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/cmake"
-    DESTINATION "${CMAKE_INSTALL_PREFIX}" MESSAGE_NEVER)
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+    ${CMAKE_BINARY_DIR}/cmake/MaterialXConfigVersion.cmake
+    VERSION ${CMAKE_VERSION}
+    COMPATIBILITY AnyNewerVersion
+)
+
+# Package for using the developper build
+export(
+    EXPORT MaterialX
+    FILE ${CMAKE_BINARY_DIR}/cmake/MaterialXConfig.cmake
+    )
```

Comment:
Gathers all the info for targets with an EXPORT tag (see other CMakeLists.txt) and write it down as a CMake package.